keras_eval/eval.py

```diff
@@ -117,14 +118,20 @@ def evaluate(self, data_dir=None, top_k=1, filter_indices=None, confusion_matrix
             top_k: The top-k predictions to consider. E.g. top_k = 5 is top-5 preds
             filter_indices: If given take only the predictions corresponding to that indices to compute metrics
             confusion_matrix: True/False whether to show the confusion matrix
+            data_augmentation: It is a dictionary consisting of 3 elements, 'scale_sizes' could be 'default' - similar
+            to Google's paper or a list of sizes. Each scaled image then will be cropped in three square parts,
+            'transforms' is a list of transforms to apply to these crops in addition to not applying any transform 
+            ('horizontal_flip', 'vertical_flip', 'rotate_90', 'rotate_180', 'rotate_270' are supported now) and
+            'crop_original' that allows to center crop the original image prior do the rest of transforms and
+            scale + croppings.
```

Comment:
May we add an example at the end?
```suggestion
            scale + croppings.
            
            E.g. data_augmentation={'scale_sizes':'default', 'transforms':['horizontal_flip', 'rotate_180'],
    'crop_original':'center_crop'}
```